macos/FluentUI/Button/Button.swift

```diff
@@ -424,7 +424,7 @@ class ButtonCell: NSButtonCell {
 			y += CGFloat(yOffsetSign) * (titleSize.height + titleToImageSpacing - titleToImageVerticalSpacingAdjustment) / 2
 		}
 
-		return NSRect(x: x, y: y, width: imageSize.width, height: imageSize.height)
+		return NSRect(x: floor(x), y: floor(y), width: ceil(imageSize.width), height: ceil(imageSize.height))
```

Comment:
I would consider using `x.rounded(.towardZero)` instead. 
Coordinates values can be negative. Consider using [rounded](https://developer.apple.com/documentation/swift/double/2886735-rounded) or round methods instead